### PR TITLE
[14.0] shopfloor_base + mobile: configure languages via backend

### PR DESCRIPTION
**Pull request by simahawk**
You can now select on each shopfloor.app the default lang and the available langs.

### CODE REVIEW

File: shopfloor_mobile_base/static/wms/src/i18n/i18n.fr.js [hunk 1]
```diff
@@ -9,11 +9,12 @@ import {translation_registry} from "../services/translation_registry.js";
...
     screen: {
         home: {
-            title: "Barcode scanner",
+            title: "Home",
+            main_title: "Home",
         },
         scan_anything: {
             name: "Scanner",
-            title: "Scanner {what}",
+            title: "Scan {what}",
```
Comment (jbaudoux) [inline shopfloor_mobile_base/static/wms/src/i18n/i18n.fr.js]: Scannez

File: shopfloor_mobile_base/static/wms/src/i18n/i18n.fr.js [hunk 2]
```diff
@@ -9,11 +9,12 @@ import {translation_registry} from "../services/translation_registry.js";
 const messages_fr = {
     screen: {
         home: {
-            title: "Barcode scanner",
+            title: "Home",
```
Comment (jbaudoux) [inline shopfloor_mobile_base/static/wms/src/i18n/i18n.fr.js]: Accueil

File: shopfloor_mobile_base/static/wms/src/i18n/i18n.fr.js (hunk 1 above)
Comment (jbaudoux) [reply shopfloor_mobile_base/static/wms/src/i18n/i18n.fr.js]: Passer en plein écran
Quitter le plein écran

**Review by jbaudoux — APPROVED** [review]
LGTM. No tested.

Once you login, do you get the language of your user?

**Comment by simahawk** [comment]
> LGTM. No tested.
> 
> Once you login, do you get the language of your user?

Good point, I added this too :wink:

**Pull request merged**